### PR TITLE
SNOW-552325 Add repo_meta.yaml

### PR DESCRIPTION
**Pull request by sfc-gh-mrui**

### CODE REVIEW

File: .github/repo_meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# point_of_contact: the owner of this repository, can be a GitHub user or GitHub team
+point_of_contact: sfc-gh-mrui sfc-gh-bli sfc-gh-ema sfc-gh-sshankar
```
Comment (sfc-gh-mrui) [inline .github/repo_meta.yaml:2]: Hi, Nathan, Can we add multiple user as `point_of_contact`? If so, how to do that?

File: .github/repo_meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+production: true
+
+# distributed: whether any source code in this repository is distributed directly to customers
+# (e.g. driver and frontend software)
+distributed: true
+
+# modified: whether any open source dependencies in this repository have been modified
+modified: false
+
+# release_branches: list of release branch patterns, exact matches or regex is acceptable
```
Comment (sfc-gh-mrui) [inline .github/repo_meta.yaml:18]: Hi, Nathan,

Spark connector uses github tags for the release, not branch. How to specify it?
For example, the latest release tags are:
```
v2.10.0-spark_3.0
v2.10.0-spark_3.1
v2.10.0-spark_3.2
```

Comment (sfc-gh-mrui) [reply .github/repo_meta.yaml:18]: what about
v*-spark_*

**Comment by sfc-gh-nsfard** [comment]
@sfc-gh-mrui repo metadata files are not required for public repos. You can close this PR if you'd like

**Comment by sfc-gh-mrui** [comment]
> @sfc-gh-mrui repo metadata files are not required for public repos. You can close this PR if you'd like

@sfc-gh-nsfard Thanks for the clarification!

**Pull request closed without merging**